easycv/utils/fp16_utils.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from collections import abc
+
+import numpy as np
+import torch
+import torch.nn as nn
+from mmcv.utils import TORCH_VERSION, digit_version
+
+
+def cast_tensor_type(inputs, src_type, dst_type):
+    """Recursively convert Tensor in inputs from src_type to dst_type.
+
+    Note:
+        In v1.4.4 and later, ``cast_tersor_type`` will only convert the
```

Comment:
can we directly use this function by importing from mmcv?

Reply:
already modified